src/main/java/io/logz/logback/LogzioSender.java

```diff
@@ -307,6 +323,23 @@ private String formatMessage(ILoggingEvent loggingEvent) {
         logMessage.addProperty("logger", loggingEvent.getLoggerName());
         logMessage.addProperty("thread", loggingEvent.getThreadName());
 
+        if (hostname != null) {
+            logMessage.addProperty("hostname", hostname);
+        }
+        if (additionalFields != null) {
+            additionalFields.forEach((k,v) -> {
```

Comment:
Thought about it, but the environment variables can change from log to log. that is why i put it there